lib/services/saml.go

```diff
@@ -181,7 +181,8 @@ func GetSAMLServiceProvider(sc SAMLConnector, clock clockwork.Clock) (*saml2.SAM
 	}
 
 	// adfs specific settings
-	if sc.GetAudience() == teleport.ADFS {
+	if sc.GetProvider() == teleport.ADFS {
+		log.Debug("[SAML] setting ADFS values")
```

Comment:
Let's use component logging here.

Reply:
He said he'd let me know when he tests it. It was a very simple typo, so I'm confident that this will work now.

Reply:
Did I add that correctly?